quickstart/csharp/dotnet/from-file/helloworld/helloworld.csproj

```diff
@@ -75,8 +75,8 @@
     <Prefer32Bit>true</Prefer32Bit>
   </PropertyGroup>
   <ItemGroup>
-    <Reference Include="Microsoft.CognitiveServices.Speech.csharp, Version=1.7.0.28, Culture=neutral, PublicKeyToken=d2e6dcccb609e663, processorArchitecture=MSIL">
-      <HintPath>..\packages\Microsoft.CognitiveServices.Speech.1.7.0\lib\net461\Microsoft.CognitiveServices.Speech.csharp.dll</HintPath>
+    <Reference Include="Microsoft.CognitiveServices.Speech.csharp, Version=1.8.0.28, Culture=neutral, PublicKeyToken=d2e6dcccb609e663, processorArchitecture=MSIL">
```

Comment:
>1.8.0.28 [](start = 75, length = 8)

should this be 1.8.0 ?

Reply:
Are you talking about the .28 ending?
If so, then no.  We started doing full Major.Minor.Patch.Build version numbers for all windows dlls, so that upgrade will work.

But we do there odd hard coded build number based on the build being release, or alpha builds.

[https://speechwiki.azurewebsites.net/architecture/carbon/current/versioning.html](https://speechwiki.azurewebsites.net/architecture/carbon/current/versioning.html)



---
In reply to: [344384593](https://github.com/Azure-Samples/cognitive-services-speech-sdk/pull/429#discussion_r344384593) [](ancestors = 344384593)